Parsers/Alsid/afad_parser.kql

```diff
@@ -0,0 +1,97 @@
+//
+//
+let CodenameTable=datatable(Codename: string, Explanation: string) [
+"test-checker-codename", "This is a test checker",
+"", "Not an alert",
+"C-ADM-ACC-USAGE", "Recent use of the default administrator account",
+"C-UNCONST-DELEG", "Dangerous delegation",
+"C-PASSWORD-DONT-EXPIRE", "Accounts with never expiring passwords",
+"C-USERS-CAN-JOIN-COMPUTERS", "Users allowed to join computers to the domain",
+"C-CLEARTEXT-PASSWORD", "Potential clear-text password",
+"C-PROTECTED-USERS-GROUP-UNUSED", "Protected Users group not used",
+"C-PASSWORD-POLICY", "Weak password policies are applied on users",
+"C-GPO-HARDENING", "Domain without computer-hardening GPOs",
+"C-LAPS-UNSECURE-CONFIG", "Local administrative account management"
+"C-AAD-CONNECT", "Verify permissions related to AAD Connect accounts",
+"C-AAD-SSO-PASSWORD", "Verify AAD SSO account password last change",
+"C-GPO-SD-CONSISTENCY", "Verify sensitive GPO objects and files permissions",
+"C-DSHEURISTICS", "Domain using a dangerous backward-compatibility configuration",
+"C-DOMAIN-FUNCTIONAL-LEVEL", "Domains have an outdated functional level",
+"C-DISABLED-ACCOUNTS-PRIV-GROUPS", "Disabled accounts in privileged groups",
+"C-DCSHADOW", "Rogue domain controllers",
+"C-DC-ACCESS-CONSISTENCY", "Domain controllers managed by illegitimate users",
+"C-DANGEROUS-TRUST-RELATIONSHIP", "Dangerous trust relationship",
+"C-DANGEROUS-SENSITIVE-PRIVILEGES", "Dangerous sensitive privileges",
+"C-DANG-PRIMGROUPID", "User Primary Group ID",
+"C-BAD-PASSWORD-COUNT", "Brute-force attack detection",
+"C-ADMINCOUNT-ACCOUNT-PROPS", "AdminCount attribute set on standard users",
+"C-ACCOUNTS-DANG-SID-HISTORY", "Accounts having a dangerous SID History attribute",
+"C-ABNORMAL-ENTRIES-IN-SCHEMA", "Dangerous rights in AD's schema",
+"C-GPOLICY-DISABLED-UNLINKED", "Unlinked, disabled or orphan GPO",
+"C-KERBEROS-CONFIG-ACCOUNT", "Kerberos configuration on user account",
+"C-KRBTGT-PASSWORD", "KDC password last change",
+"C-LAPS-UNSECURE-CONFIG", "Local administrative account management",
+"C-NATIVE-ADM-GROUP-MEMBERS", "Native administrative group members",
+"C-NETLOGON-SECURITY", "Unsecured configuration of Netlogon protocol",
+"C-OBSOLETE-SYSTEMS", "Computers running an obsolete OS",
+"C-PASSWORD-NOT-REQUIRED", "Account that might have an empty password",
+"C-PKI-WEAK-CRYPTO", "Use of weak cryptography algorithms into Active Directory PKI",
+"C-PRE-WIN2000-ACCESS-MEMBERS", "Accounts using a pre-Windows 2000 compatible access control",
+"C-PRIV-ACCOUNTS-SPN", "Privileged accounts running Kerberos services",
+"C-REVER-PWD-GPO", "Reversible passwords in GPO",
+"C-ROOTOBJECTS-SD-CONSISTENCY", "Root objects permissions allowing DCSync-like attacks",
+"C-SDPROP-CONSISTENCY", "Ensure SDProp consistency",
+"C-SENSITIVE-CERTIFICATES-ON-USER", "Ensure SDProp consistency",
+"C-SLEEPING-ACCOUNTS", "Sleeping accounts",
+"C-USER-PASSWORD", "User account using old password",
+"C-USERS-REVER-PWDS", "Reversible passwords",
+];
+
```

Comment:
Remove space on this line (line number 66), otherwise, CodenameTable will not be recognized.